### PR TITLE
Improve readme and contributor process

### PR DESCRIPTION
**Pull request by serinamarie**
- Detail the contributor process and update the `README` to include examples.

- Make the PR template more succinct

Any feedback or ways to improve is appreciated!

### CODE REVIEW

**Review by gabcoyne — APPROVED** [review]

**Review by robfreedy — APPROVED** [review]
Chef marvin is fire

**Review by robfreedy — DISMISSED** [review]

**Comment by serinamarie** [comment]
> Chef marvin is fire

Terry is a dall-e artist

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -9,8 +10,8 @@ Please include a summary of the feature/fix. If this PR references an issue, ple
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] This change requires a documentation update
 
-## Checklist
-<!-- If you are adding a new recipe: -->
+## New Recipe Checklist
+<!-- If adding a new recipe, please ensure this list is completed. -->
 - [ ] My PR is in the format of `Add project-name recipe`
```
Comment (zzstoatzz) [inline .github/pull_request_template.md:15]:
```suggestion
- [ ] My PR is in the format of `Add <project-name> recipe`
```

File: README.md [hunk 2]
```diff
@@ -1,82 +1,57 @@
-<div id="top"></div>
 
-<!-- PROJECT LOGO -->
-<br />
+# Prefect Recipes 🧑‍🍳 🥐
+
+This repository contains common and extensible Prefect patterns to drive efficient workflows. We like to call these patterns our recipes. Here you'll find starter code and more advanced example use cases!
```
Comment (zzstoatzz) [inline README.md:4]:
```suggestion
This repository contains common and extensible Prefect patterns to drive efficient workflows. We like to call these patterns our **recipes**. Here you'll find starter code and more advanced example use cases!
```

Comment (zzstoatzz) [inline README.md:4]:
```suggestion
This repository contains common and extensible Prefect patterns to drive efficient workflows - we like to call these patterns our **recipes**!

Here you'll find starter code and more advanced example use cases.
```

**Review by zzstoatzz — APPROVED** [review]

**Review by zzstoatzz — APPROVED** [review]
LGTM!!

**Pull request merged**